### PR TITLE
pkp/pkp-lib#862 including logging for webservice requests to error log

### PR DESCRIPTION
**Pull request by jnugent**
Include logging if the config option is enabled in the parent application.

### CODE REVIEW

File: classes/webservice/XmlWebService.inc.php [hunk 1]
```diff
@@ -63,7 +63,9 @@ function setReturnType($returnType) {
 	function &call(&$webServiceRequest) {
 		// Call the web service
 		$xmlResult = parent::call($webServiceRequest);
-
+		if (Config::getVar('debug', 'log_web_service_info')) {
+			error_log('Time: ' . date('c') . "\nRequest: " . print_r($webServiceRequest, true) . "\nResponse: " . print__r($xmlResult, true) . "\nLast response status: " . $this->_lastResponseStatus . "\n", 0);
```
Comment (asmecher) [inline classes/webservice/XmlWebService.inc.php]: That looks like a typo -- `print__r`

Comment (jnugent) [inline classes/webservice/XmlWebService.inc.php]: Whoops, fixed!

File: classes/webservice/XmlWebService.inc.php [hunk 2]
```diff
@@ -63,7 +63,9 @@ function setReturnType($returnType) {
 	function &call(&$webServiceRequest) {
 		// Call the web service
 		$xmlResult = parent::call($webServiceRequest);
-
+		if (Config::getVar('debug', 'log_web_service_info')) {
+			error_log('Time: ' . date('c') . "\nRequest: " . print_r($webServiceRequest, true) . "\nResponse: " . print_r($xmlResult, true) . "\nLast response status: " . $this->_lastResponseStatus . "\n", 0);
```
Comment (asmecher) [inline classes/webservice/XmlWebService.inc.php]: Mind also removing the `0` magic number from the call to `error_log`? It's implicit and I hate magic numbers.

Comment (jnugent) [inline classes/webservice/XmlWebService.inc.php]: Also done!

**Pull request merged**